packages/Syntax/src/Syntax.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import Syntax from '.';
+import { Lang } from './types';
+
+const codeSnippet = `\
+const insertDocuments = function(db, callback) {
+  // Get the documents collection
+  const collection = db.collection('documents');
+  // Insert some documents
+  collection.insertMany([
+    {a : 1}, {a : 2}, {a : 3}
+  ], function(err, result) {
+    assert.equal(err, null);
+    assert.equal(3, result.result.n);
+    assert.equal(3, result.ops.length);
+    console.log("Inserted 3 documents into the collection");
+    callback(result);
+  });
+}
+
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
+
+// Connection URL
+const url = 'mongodb://localhost:27017';
+
+// Database Name
+const dbName = 'myproject';
+const client = new MongoClient(url);
+
+// Use connect method to connect to the server
+client.connect(function(err) {
+  assert.equal(null, err);
+  console.log("Connected successfully to server");
+
+  const db = client.db(dbName);
+
+  insertDocuments(db, function() {
+    client.close();
+  });
+});
+`;
+
+storiesOf('Syntax', module).add('Syntax', () => (
+  <Syntax lang={select('language', Lang, 'auto')}>{codeSnippet}</Syntax>
```

Comment:
Can we make the codeSnippet customizable as well? Does storybook support multiline strings?

Reply:
I added a couple snippets to the code storybook. Making it a textarea I had issues with though. Could try again...

Reply:
<img width="351" alt="Screen Shot 2019-07-01 at 4 17 42 PM" src="https://user-images.githubusercontent.com/25213299/60471972-cedab300-9c1b-11e9-83bd-518495d9e86b.png">
That's right – storybook's `text()` loves adding escape characters...

Reply:
Looks like you can disable that: https://github.com/storybookjs/storybook/issues/4445

Reply:
😮Woo! Will add this!